modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/MysqlSchemaCodegen.java

```diff
@@ -364,7 +364,9 @@ public void processIntegerTypeProperty(CodegenModel model, CodegenProperty prope
             List<Object> enumValues = (List<Object>) allowableValues.get("values");
             for (int i = 0; i < enumValues.size(); i++) {
                 if (i > ENUM_MAX_ELEMENTS - 1) {
-                    LOGGER.warn("ENUM column can have maximum of " + ENUM_MAX_ELEMENTS.toString() + " distinct elements, following value will be skipped: " + (String) enumValues.get(i));
+                    LOGGER.warn(
+							"ENUM column can have maximum of {} distinct elements, following value will be skipped: {}",
+							ENUM_MAX_ELEMENTS.toString(), (String) enumValues.get(i));
```

Comment:
@cal101 @agilob not sure why these spaces are added after the refactor. Will file a PR to clean it up manually later instead.

Reply:
toString also should be removed from refactored code as LOGGER will call that internally